build.gradle

```diff
@@ -19,41 +20,109 @@ ext.githubProjectName = rootProject.name // Change if github project name is not
 apply plugin: 'project-report'
 apply from: "$rootDir/versionsOfDependencies.gradle"
 
+allprojects {
+    apply plugin: 'idea'
+    apply plugin: 'jacoco'
+    apply plugin: 'eclipse'
```

Comment:
why do we need eclipse and idea plugin?

Reply:
we don't but I assume other users might use both